packages/@tinacms/fields/package.json

```diff
@@ -59,7 +59,9 @@
     "color-string": "^1.5.3",
     "react-color": "^2.17.3",
     "react-dismissible": "^1.3.0",
-    "react-dropzone": "10.1.8"
+    "react-dropzone": "10.1.8",
+    "moment": ">=2 <=3",
```

Comment:
https://day.js.org/ seems to be much lighter and have the same moment API

Reply:
I'll have to take a look at that; not heard of `day.js`.  Been looking at `date-fns` (which seems to be where most of the community went with their `datepicker` solutions) and `luxom` (which shares one of the creators/maintainers of Moment.js).

Ideally, we'd choose a `datepicker` solution that uses a minimal `date` library and use that same `date` library in the gateway so that parsing/formating is the same on the frontend and backend (right now, it isn't).

Reply:
I saw `day.js` is used on https://github.com/calendso/calendso an open source Calendly alternative who is built with Next.js that have to rely on a good date picker with timezone support since it's core to their product.

Reply:
Never heard of it, but it definitely checks all the boxes of something we might want for an alternative.  @spbyrne Have you heard of https://github.com/calendso/calendso in your circles?  I'm going to take a closer look and see what functionality it offers vs. `react-datepicker`.